test/Constants.cs

```diff
@@ -0,0 +1,16 @@
+namespace Ovh.Test
+{
+    public static class Constants
+    {
+        public const string ENDPOINT = "ovh-eu";
+        public const string APPLICATION_KEY = "APPLICATION_KEY";
+        public const string APPLICATION_SECRET = "APPLICATION_SECRET";
+        public const string CONSUMER_KEY = "CONSUMER_KEY";
+        public const string OVH_APP_HEADER = "X-Ovh-Application";
+        public const string OVH_CONSUMER_HEADER = "X-Ovh-Consumer";
+        public const string OVH_TIME_HEADER = "X-Ovh-Timestamp";
+        public const string OVH_SIGNATURE_HEADER = "X-Ovh-Signature";
+        public const string OVH_BATCH_HEADER = "X-Ovh-Batch";
+    }
```

Comment:
You can move this constants class directly on the `Client` part of the code : this may improve consistency between production code and tests?

Reply:
Yeah, I probably started by the testing ones, then added these without thinking. Changed